apps/src/templates/progress/progressHelpers.js

```diff
@@ -16,6 +22,8 @@ import {LevelStatus, LevelKind} from '@cdo/apps/util/sharedConstants';
 export function lessonIsVisible(lesson, state, viewAs) {
   if (!viewAs) {
     throw new Error('missing param viewAs in lessonIsVisible');
+  } else if (viewAs === ViewType.Teacher) {
```

Comment:
If viewAs == ViewType.Teacher and (lesson.lockable && !state.stageLock.lockableAuthorized), this function will now return true instead of false.  Just wanted to double-check that that change was intentional?

Reply:
good catch! i moved this early return to after the lockable/lockableAuthorized check.